README.md

```diff
@@ -1,24 +1,52 @@
-Terraform Provider for Google Cloud Platform
+Terraform Provider for Google Cloud Platform (Beta)
 ==================
 
 - Website: https://www.terraform.io
+- Tutorials: [learn.hashicorp.com](https://learn.hashicorp.com/terraform?track=getting-started#getting-started)
+- Forum: [discuss.hashicorp.com](https://discuss.hashicorp.com/c/terraform-providers/tf-google/)
 - Documentation: https://www.terraform.io/docs/providers/google/index.html
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
-<img src="https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-hashicorp.svg" width="600px">
 
-Maintainers
------------
+The Terraform Google provider is a plugin for Terraform that allows for management of GCP resources.
```

Comment:
Do we want to add `Beta` in here? We leave it out in a few places throughout the doc, I have no strong preference, just wanted to mention it.